content/articles/overview-of-rootkits/index.md

```diff
@@ -0,0 +1,55 @@
+
+The malware helps the hackers to steal personal data and financial information once they have gained illegal access to computers.
+
+Although this may sound concerning, rootkits can sometimes be put to good use. For example, they are purposefully deployed for R&D to see how the system reacts to it. Then the report can be recorded, studied, and used to improve existing and future systems to avoid any future threats.
+<!-- what is R&D? Please explain -->
+
+### How it affects our systems
+Once the rootkit is on the system, it makes the operating system vague about its presence. It achieves this by avoiding standard operating system services such as detecting and tracking unusual activity and other third-party applications such as anti-virus software and anti-malware components.
+
+Capturing keystrokes is one of the destructive acts rootkits may accomplish. This may assist fraudsters in obtaining your online account credentials, which they can use to disable the online security features. This causes performance slowdown, bootloader anomalies, data loss, and a variety of other problems.
+
+### Types of rootkits
+Rootkits are classified into several types based on the part of the system they affect and how they operate. Here are some of the most concerning rootkits.
+
+#### Kernel-mode rootkits
+Rootkits that target the kernel of the operating system are called kernel-mode rootkits.
+
+The kernel is the core of the operating system, therefore a rootkit on that section equals a serious threat. The rootkit can simply add or remove a significant chunk of the operating system and its associated modules. Kernel rootkits are more difficult to detect and remove since they operate on the kernel, they are immediately regarded as vital as the operating system. In this condition, no aspect of the system can be trusted because they can now change how the operating system works as a whole.
+
+#### Bootkits
+Bootkits are a type of kernel-mode rootkit that affects the system's booting mechanism. Rootkits of this type replace the original bootloader with the infected one. This allows the hackers to access the system's startup code. This launches the rootkit even before your computer's operating system is fully loaded, resulting in rootkit execution when the machine ultimately boots up.
+
+#### Firmware rootkit
+A firmware rootkit is a type of rootkit that exploits essential hardware such as routers, drives, and so on. It takes advantage of the firmware because it is unaffected by unit testing and security software. Before deploying a software update, the system is reviewed for quality and tested to maximize code coverage. During this stage, the majority of harmful files and codes are eliminated from the system. As a result, this type of rootkit latches onto the firmware and uses it as leverage.
+
+### Preventive measures

```

Comment:
The points in this section sound important. Please convert each paragraph into a sub-heading with a description below it.